lib/main.js

```diff
@@ -28,7 +28,7 @@ define(["moment/moment", "router", "leaflet", "gui", "helper", "moment/locale/de
               dataGraph.batadv.links = dataGraph.batadv.links.concat(data[i].batadv.links);
               dataGraph.timestamp = data[i].timestamp;
             }
-          } else if (data[i].version !== 2) {
+          } else if (data[i].version !== 2 && data[i].version !== -1) {
```

Comment:
'-1' because the nodes.json would not be defined, it would be change with the change of the respondd application on gluon